tcf_website/templates/base/sidebar.html

```diff
@@ -24,11 +24,12 @@
        {% if request.resolver_match.url_name == 'about' %}active{% endif %}">
       <i class="fa fa-code fa-fw"></i>&nbsp; About
     </a>
+
+    <!-- tCF profile dropdown when navbar collapses -->
     {% if user.is_authenticated %}
-      <!-- KEEP IN SYNC WITH navbar.html -->
-      <div class="d-sm-none list-group-item dropdown">
+      <div class="d-md-none list-group-item dropdown">
         <div class="dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-        <i class="fa fa-user fa-fw"></i>&nbsp; Hi, {{user.first_name}}
+        <i class="fa fa-user fa-fw"></i>&nbsp; Hey {{user.first_name}}!
```

Comment:
extreme nitpick: I think "Hey, Bradley!" is correct grammar

Reply:
hey is for horses >:(

Reply:
tough crowd 🙄

Reply:
fixed the grammer